reference_system_autoware/README.md

```diff
@@ -0,0 +1,74 @@
+# reference_system_autoware
+
+This file is meant to define the Autoware Reference System and all of its nodes, topics and message types.
+
+![Node graph of reference-system-autoware](../content/img/dotgraph_autoware.svg)
+
+## Message Types
+
+A **single message type** is used for the entire _reference system_ when generating results in order to simplify the setup as well as make it more repeatible and extensible.
+
+This means **only one _message type_** from the list below is used during any given experimental run for every node in the reference system.
+
+1. [**Message1kB**](../reference_interfaces/msg/Message1kB.idl)
+    - reference message with a fixed size of 1 kilobyte (kB)
+
+Other messages with different fixed sizes could be added here in the future.
+
+When reporting results it will be important to include the _message type_ used duing the experiement so that comparisons can be done "apples to apples" and not "apples to pears".
+
+## Reference System Autoware
+
+Built from [a handful of building-block node types](../README.md#concept-overview), each one of these nodes are meant to simulate a real-world node from the Autoware.Auto project lidar data pipeline.
+
+Under each node type are the requirements used for _this specific reference system_, `reference_system_autoware`. Future reference systems could have slightly different requirements and still use the same building-block node types.
+
+For simplicity's sake, every node except for the _command nodes_ only ever publishes one topic and this topic has the same name as the node that publishes it. However, each topic can be subscribed to by multiple different nodes.
+
+1. [**Message Type**](#message-types)
+    - all nodes use the same message type during any single test run
+    - default _message type_:
+        - [Message1kB](include/reference_system_autoware/types.hpp#L21)
+2. [**Sensor Nodes**](reference_system_autoware/include/reference_system_autoware/node/sensor.hpp)
+    - all _sensor nodes_ have a publishing rate (cycle time) of [**100 milliseconds**](include/reference_system_autoware/reference_system.hpp#L39)
```

Comment:
@elfenpiff whatever we end up using for the processing times for each node type we should update these values here in the requirements